### PR TITLE
EosStation: Testing 

### PR DESCRIPTION
**Pull request by EIB0N**
## About The Pull Request

This adds EosStation the map, it isn't finished but it is about 99% done this PR is meant to have it be selectable for TESTING.

SECURITY:

EosStation_Security

COMMAND:

EosStation_Command

ARRIVAL/EVAC:

EosStation_Arrival Evac

SCIENCE:

EosStation_Science

SERVICE:

EosStation_Service

ENGINEERING/ ATMOSPHERICS

EosStation_Engineering_Atmospheric

MEDBAY:

EosStation_Medbay

SUPPLY

EosStation_Supply

A.I

EosStation_AI

## Why It's Good For The Game

New Map more fun

## Changelog
:cl:
add: EosStation

/:cl:

### CODE REVIEW

File: _maps/eosstation.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+	"version": 1,
+	"map_name": "EosStation",
+	"map_path": "map_files/EosStation",
+	"map_file": "Eosstation.dmm",
```
Comment (SgtHunk) [inline _maps/eosstation.json:5]:
```suggestion
	"map_path": "map_files/Eosstation",
	"map_file": "EosStation.dmm",
```

**Review by SgtHunk — CHANGES_REQUESTED** [review]
This needs to be part of the basemap #include

File: _maps/eosstation.json (hunk 1 above)
Comment (EIB0N) [reply _maps/eosstation.json]: changed

Comment (SgtHunk) [reply _maps/eosstation.json]: The map_file still isn't changed
tbh should just rename the folder to EosStation for consistency

**Comment by JohnFulpWillard** [comment]
Is the dining area its own area like on other maps? If so, wouldnt it be like this
image

File: _maps/_basemap.dm [hunk 2]
```diff
@@ -14,7 +14,7 @@
 		#include "map_files\HelioStation\HelioStation.dmm"
 		#include "map_files\tramstation\tramstation.dmm"
 		#include "map_files\SeleneStation\SeleneStation.dmm"
-
+		#include "map_files\EosStation\EosStation.dmm"
 		#ifdef CIBUILDING
```
Comment (SgtHunk) [inline _maps/_basemap.dm:18]:
```suggestion
		#include "map_files\EosStation\EosStation.dmm"

 		#ifdef CIBUILDING
```
Muh space

**Comment by remove32** [comment]
theres no way this bitch dont crash

File: fulp_modules/maps/turfs/closed.dm [hunk 3]
```diff
@@ -0,0 +1,6 @@
+/// Used in EosStation.
```
Comment (Fikou) [inline fulp_modules/maps/turfs/closed.dm:1]: you can just set the baseturf in the map json file. theres no need for this

File: _maps/_basemap.dm [hunk 4]
```diff
@@ -13,9 +13,10 @@
 		#include "map_files\IceBoxStation\IceBoxStation.dmm"
 		#include "map_files\HelioStation\HelioStation.dmm"
 		#include "map_files\SeleneStation\SeleneStation.dmm"
+		#include "map_files\EosStation\EosStation.dmm"
 		#include "map_files\tramstation\tramstation.dmm"
 
-		#ifdef CIBUILDING
+#ifdef CIBUILDING
```
Comment (Fikou) [inline _maps/_basemap.dm:19]: why the indent change?

**Comment by remove32** [comment]
Update EosStation.dmm

**Comment by ModDownloading** [comment]
Well, attempting to revive Cerestation in an improved form already makes me happy.

~~REPLACE ALL THE BRIDGES WITH TRAMS, NO WAY IT ENDS BADLY~~

**Comment by EIB0N** [comment]
> Well, attempting to revive Cerestation in an improved form already makes me happy.
> 
> ~REPLACE ALL THE BRIDGES WITH TRAMS, NO WAY IT ENDS BADLY~

ty brother also depends if the server can even handle such a map

**Comment by remove32** [comment]
Update EosStation.dmm

**Comment by GaryGreytide** [comment]
Big Chungus Station

**Comment by redguy999** [comment]
There's a floor missing nearing the chapel, hidden under a maint airlock.

**Comment by EIB0N** [comment]
Welp I'm crazy lost in the sauce so ill just make a new one; neglected updating it for too long

**Comment by EIB0N** [comment]
ok one more attempt

**Pull request closed without merging**